### ISSUE TITLE
MonthCalendar control - Narrator is not announcing more information about control

### ISSUE THREAD

**Issue by M-Lipin** [dotnet/winforms#1911]
**Repro Steps:​**
1. Launch .NET Core WinForms app with MonthCalendar control.
2. Start Narrator.
3. Verify that narrator is announcing the detailed information about calendar control or not.

**Actual:**
Narrator is not announcing the more information about Month Calendar.

**Expected:**
When narrator focus shifts on Month Calendar narrator should announce the selected date in proper format like dd/mm/yy. 

**User Impact:**
Screen reader users will face difficulty understand the month calendar properties.

**Comment by M-Lipin** [comment]
DevDiv issue: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/956404

**Comment by M-Lipin** [comment]
@merriemcgaw the fix implemented in #2090 implies changing MonthCalendar accessible control type to **calendar** like this is in WPF. One exception is the case when the label is added to the form just before MonthCalendar control (as a title for calendar): in this particular scenario, I've found out that if calendar control type is used here, Narrator does not announce the custom name of the control defined by this label, so for this specific case I'm keeping the MonthCalendar control type as **table**. It seems to me that this is acceptable and does not break things as table accessible control type was used before my changes.

**Comment by M-Lipin** [comment]
Resolving this issue as the fixing PR #2090 is merged to release/3.1.

**Comment by John-Qiao** [comment]
Verified with latest .NET Core 5.0 5.0.100-alpha.1.20064.2 build, it works as Mikhail's description.
![TestResult-1911](https://user-images.githubusercontent.com/45864985/72415114-bf354d00-37ae-11ea-99eb-348b83e412c4.png)
